packages/colors/README.md

```diff
@@ -1,11 +1,49 @@
-# `@livechat/design-system-colors`
+# @livechat/design-system-colors
 
-> TODO: description
+> Colors package for apps using LiveChat Design System
+
+## Getting started
+
+To install `@livechat/design-system-colors` in your project, you will need to run the following
+command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install @livechat/design-system-colors --save
+```
 
 ## Usage
 
+There are three ways to use LiveChat design colors palette:
+- use JavaScript implementation of colors
 ```
-const colors = require('colors');
+import Colors from '@livechat/design-system-colors'
+```
+- import `design-system-colos.css` file and style UI elements with classes
```

Comment:
I decided to declare vars on `:root` instead of a special class. Although I don't find useful using additional, special class, I remember we talked about 'cascade' property of CSS styles. Could you please remind me why it was important in this case?